### ISSUE TITLE
TypeError: i is not a function. (In 'i(alpine_default)', 'i' is an instance of Object)

### ISSUE THREAD

**Issue by shaunrobinson** [oli-laban/alpine-tailwind-lightbox#1]
This is exactly what I am looking for but it doesn't work for me.

Installing with npm and including it in the scripts file I get:

```
TypeError: i is not a function. (In 'i(alpine_default)', 'i' is an instance of Object)
```

Does this mean anything to you?

**Comment by oli-laban** [comment]
@shaunrobinson It's not something I've come across. Could you post how you're including Alpine + the plugin, or reproduce in CodePen/JSFiddle?

**Comment by shaunrobinson** [comment]
I'm using the _tw theme for WordPress: https://underscoretw.com/

Other than that, I think I'm just including it normally, as per the instructions. I'm also using the following plugins, which all work well (and I did try removing some just in case there were conflicts, but there didn't seem to be any):

import Alpine from 'alpinejs';
import collapse from '@alpinejs/collapse';
import focus from '@alpinejs/focus';
import lightbox from 'alpine-tailwind-lightbox';
import Tooltip from '@ryangjchandler/alpine-tooltip';

Alpine.plugin(focus);
Alpine.plugin(collapse);

Alpine.plugin(Tooltip);
Alpine.plugin(lightbox);
window.Alpine = Alpine;

**Comment by oli-laban** [comment]
@shaunrobinson The package.json was pointing esbuild to the CDN build. Just published v1.0.2 with a fix. Sorry about that!

**Comment by shaunrobinson** [comment]
That fixed it, thanks!
